test/specs/addons/Textarea-test.js

```diff
@@ -10,10 +10,6 @@ describe('Textarea', () => {
     var renderedTextarea = render(<Textarea name='sample-post' />);
     expect(renderedTextarea.first().props.name).to.equal('sample-post');
   });
-  it('has 3 rows by default', () => {
-    var renderedTextarea = render(<Textarea name='sample' />);
-    expect(renderedTextarea.findTag('textarea').props.rows).to.equal('3');
-  });
```

Comment:
We don't set arbitrary styling defaults.  This will be left to the users of Stardust.  There will be future PRs to remove other styling and defaults as well as document any magic we are doing.